client/my-sites/plugins/plugin-details-CTA/index.jsx

```diff
@@ -217,6 +221,15 @@ const CTAButton = ( {
 	);
 	const hasPreferences = useSelector( hasReceivedRemotePreferences );
 
+	const primaryDomain = useSelector( ( state ) =>
+		getPrimaryDomainBySiteId( state, selectedSite?.ID )
+	);
+
+	const pluginRequiresCustomPrimaryDomain =
+		( primaryDomain?.isWpcomDomain || primaryDomain?.isWpcomStagingDomain ) &&
```

Comment:
`isWpcomDomain` should be `isWPCOMDomain` to match the case. Currently this is failing for simple sites.

Reply:
Thanks for pointing this out! 🎉